### PR TITLE
Exp now updates every minute

### PR DESCRIPTION
**Pull request by Cyberboss**
As opposed to 10 minutes

:cl:
fix: Resolution for job EXP tracking increased to 1 minute from 10
/:cl:

Also pop count

This fixes people not getting up to 9 minutes of EXP when they die

### CODE REVIEW

**Comment by epochayur** [comment]
is EXP actually used for anything?

**Comment by MrStonedOne** [comment]
@Shadowlight213

**Comment by Cyberboss** [comment]
@epochayur it restricts head roles and antag leaders

**Review by Shadowlight213 — CHANGES_REQUESTED** [review]
You need to change `update_exp(10,FALSE)` on line 35 to
`update_exp(1,FALSE)`.
Otherwise it adds 10 minutes of time every one minute.

**Review by Shadowlight213 — APPROVED** [review]

File: code/controllers/subsystem/blackbox.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@ SUBSYSTEM_DEF(blackbox)
 
 	if(CONFIG_GET(flag/use_exp_tracking))
 		if((triggertime < 0) || (world.time > (triggertime +3000)))	//subsystem fires once at roundstart then once every 10 minutes. a 5 min check skips the first fire. The <0 is midnight rollover check
```
Comment (MrStonedOne) [inline code/controllers/subsystem/blackbox.dm]: time value would need to be changed as well. (or removed since it would have less of an effect)

**Comment by optimumtact** [comment]
@Cyberboss do the needful

**Comment by KorPhaeron** [comment]
@Cyberboss stale, reopen when you're done

**Comment by Cyberboss** [comment]
I forgot

**Comment by MrStonedOne** [comment]
Hmm. do you think you could make that track time since last update and dynamically calculate how much to apply. or at least calculate it based off of wait. 

I ask because we've had issues on terry where the uk<->us network connection went all fucky and added lag to queries. this way if it ever goes haywire again on terry we can slow down the fire rate to reduce lag by var-editing the wait var. ('member, in byond waiting for a query to update is a hard block) (and this way if needed I could slow down the fire rate on terry with a compile extension that hooked into blackbox's New())

**Comment by AnturK** [comment]
I'm kinda hesitant to merge this with current terry sql issues.

**Comment by Cyberboss** [comment]
I've begun to hate this myself too

**Pull request closed without merging**